docs/cloud/webhooks.md

```diff
@@ -0,0 +1,277 @@
+Because this webhook will only be used for a single event from a single resource, your
+template can be entirely static:
+
+```JSON
+{
+    "event": "model.refreshed",
+    "resource": {
+        "prefect.resource.id": "product.models.recommendations",
+        "prefect.resource.name": "Recommendations [Products]",
+        "producing-team": "Data Science"
+    }
+}
+```
+
+!!! tip "Make sure to produce valid JSON"
+
+    The output of your template, when rendered, should be a valid string that can be
+    parsed, for example, with `json.loads`.
+
+A webhook with this template may be invoked via _any_ of the HTTP methods, including a
+`GET` request with no body, so the team you are integrating with can include this at the
+end of their daily script:
+
+```bash
+curl https://api.prefect.cloud/hooks/AERylZ_uewzpDx-8fcweHQ
+```
+
+Each time the script "pings" the webhook like this, the webhook will produce a single
+Prefect event with that name and resource in your workspace.
+
+### Event fields that Prefect Cloud populates for you
+
+You may notice that you only had to provide the `event` and `resource` definition, which
+is not a completely fleshed out event.  Prefect Cloud will set default values for any
+missing fields, like `occurred` and `id`, so that you don't need to set them in your
+template.  Additionally, Prefect Cloud will always add the webhook itself as a related
+resource on all of the events it produces.
+
+If your template does not produce a `payload` field, the `payload` will default to a
+standard set of debugging information, including the HTTP method, headers, and body.
+
+### Dynamic webhook events
+
+Now let's say that after a few days you and the Data Science team are getting a lot of
+value from the automations you have set up with the static webhook.  You've agreed to
+upgrade this webhook to handle all of the various models that they produce.  It's time
+to add some dynamic information to your webhook template.
+
+Your colleages in Data Science have adjusted their daily `cron` scripts to `POST` a
+small body that includes the ID and name of the model that was just refreshed:
+
+```bash
+curl -d "model=recommendations" -d "friendly_name=Recommendations%20[Products]" -X POST https://api.prefect.cloud/hooks/AERylZ_uewzpDx-8fcweHQ
+```
+
+This means that they are sending a `POST` request and the body will include a
+traditional URL-encoded form with two fields `model` and `friendly_name` describing the
+model that was just refreshed.  To receive these in your template and produce different
+events for the different models:
+
+```jinja2
+{
+    "event": "model.refreshed",
+    "resource": {
+        "prefect.resource.id": "product.models.{{ body.model }}",
+        "prefect.resource.name": "{{ body.friendly_name }}",
+        "producing-team": "Data Science"
+    }
+}
+```
+
+All of the subsequent `POST` requests will start to produce events with those variable
```

Comment:
what if i have a template like this and I dont want to pass any data, can i specify a server side default? maybe with jinjas default filter? 
  Hey {{ first_name|default("there", true) }},

Reply:
Ohhhhh that's slick, I'll find a good place to mention that.